.gitlab/source_test/ebpf.yml

```diff
@@ -19,6 +19,7 @@
   before_script:
     - !reference [.retrieve_linux_go_deps]
     - !reference [.retrieve_linux_go_tools_deps]
+    - source /root/.bashrc && conda activate ddpy3
```

Comment:
Could you edit the `entrypoint-sysprobe.sh` file in the buildimages to avoid the need for this? Seems like something that is easy to forget.

Also I think `/root/.bashrc` already calls `conda activate ddpy3` right?

Reply:
I was told the entrypoint won't be traversed in the k8s runners. I think that's why this line of code is called so broadly in the CI codebase:
<img width="561" alt="image" src="https://github.com/DataDog/datadog-agent/assets/47357713/26238969-e720-4ca5-9c3f-7ab2a40594d7">
So I'm not sure this would work. I can try though if you want !

Reply:
> Also I think /root/.bashrc already calls conda activate ddpy3 right?

Might be the case, I'll check this on monday !

Reply:
Double checked it, and the `conda activate ddpy3` is indeed already in the `/root/.bashrc` file. Removing it, thanks !

Reply:
Fixed with [d68adc9](https://github.com/DataDog/datadog-agent/pull/23773/commits/d68adc9a1d468eb41aab908408a83d1a1df56df0).